frontend/src/Commons/Forms/InputModules/Crops/CropsInfo.tsx

```diff
@@ -223,19 +221,21 @@ const CropsInfoComponent: FC<InputModuleProps> = ({
                         id="cropId"
                         label="Crop"
                         options={CropIDOptions}
-                        desktopWidth="40%"
+                        desktopWidth="226px"
+                        mobileWidth="137px"
+                        CropField
                         onChange={(e) => handleChange(e, setFieldValue)}
                       />
-                      <StyledAreaContainer formCrops>
-                        <CustomField
-                          label="Yield"
-                          id="yield"
-                          name="yield"
-                          type="number"
-                          desktopWidth="50%"
-                        />
-                        <p>tons/ac</p>
-                      </StyledAreaContainer>
+                      <CustomField
+                        label="Yield"
+                        id="yield"
+                        name="yield"
+                        type="number"
+                        isYield
```

Comment:
quick response to editing a field and not affecting other fields using ternary operator